blaze-server/src/main/scala/org/http4s/blaze/server/BlazeServerBuilder.scala

```diff
@@ -240,7 +240,7 @@ class BlazeServerBuilder[F[_]] private (
     copy(selectorThreadFactory = selectorThreadFactory)
 
   @deprecated("This operation is a no-op. WebSockets are always enabled.", "0.23")
-  def withWebSockets(enableWebsockets: Boolean): Self =
+  def withWebSockets(enableWebSockets: Boolean): Self =
```

Comment:
Not sure about this change

Reply:
Because this is part of the type-signature?

Again, happy to drop now and ask questions later if it'll help.